src/timers.cc

```diff
@@ -38,6 +38,9 @@
 #include <mach/mach_port.h>
 #include <mach/thread_act.h>
 #endif
+#if defined(BENCHMARK_OS_QURT)
+#include <qurt/qurt.h>
```

Comment:
is there a difference between `<qurt.h>` and `<qurt/qurt.h>`? you use both in this change.

Reply:
Good catch! Due to how include paths are setup, both refer to the same file, but `<qurt.h>` is the canonical one. Fixed.